pytket/phir/phirgen_parallel.py

```diff
@@ -31,25 +31,26 @@
 
 
 def exec_order_preserved_helper(
-    ordered_dict: OrderedDict["UnitID", int], group_number: int, qubit_last_group: int
+    group_exec_order: list[int], group_number: int, qubit_last_group: int
 ) -> bool:
     """A helper to determine whether order is preserved when adding qubits to groups."""
     # determine whether the current group number is later in execution
     # than the last group in which a qubit was used
```

Comment:
Is this comment describing the role of this helper? If so, it should be part of the docstring.